README.md

```diff
@@ -194,6 +194,12 @@ Restricting service binding to resources within the same namespace is strongly *
 
 A Service Binding resource **MUST** define a `.spec.application` which is an `ObjectReference`-like declaration to a `PodSpec`-able resource.  A `ServiceBinding` **MAY** define the application reference by-name or by-[label selector][ls]. A name and selector **MUST NOT** be defined in the same reference.  A Service Binding resource **MUST** define a `.spec.service` which is an `ObjectReference`-like declaration to a Provisioned Service-able resource.  Extensions and implementations **MAY** allow additional kinds of applications and services to be referenced.
 
+The Service Binding resource **MAY** define `.spec.application.containers`, as a list of integers or strings, to limit which containers in the application are bound.  Binding to a container is opt-in, unless `.spec.application.containers` is undefined then all containers **MUST** be bound.  For each item in the containers list:
```

Comment:
```suggestion
The Service Binding resource **MAY** define `.spec.application.containers`, as a list of integers or strings, to limit which containers in the application are bound.  Binding to a specific container or containers is opt-in. If `.spec.application.containers` is undefined, the reconciler **MUST** bind all containers.  For each item in the containers list:
```

Reply:
@scothis Let me know if the rephrasing makes the sentence more clear.

Reply:
this sentence doesn't stand on it's own anymore:
> Binding to a specific container or containers is opt-in.

Since the default behavior is to bind all containers, that's not opt-in.

Reply:
I think that sentence can be removed because the next sentence is sufficient.

> If `.spec.application.containers` is undefined, the reconciler **MUST** bind all containers.

Reply:
Without that sentence a conforming implementation could always bind all containers and completely ignore the content of `.spec.application.containers`.

Reply:
Ok, I am withdrawing my suggestion. Thanks :+1: